Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -146,6 +146,36 @@ struct SymbolGraphLoader {
         
         self.symbolGraphs = loadedGraphs.mapValues(\.graph)
         (self.unifiedGraphs, self.graphLocations) = graphLoader.finishLoading()
+        
+        unifiedGraphs.forEach { (_, unifiedGraph) in
+            var unifiedGraph = unifiedGraph
+            var defaultUnconditionallyUnavailablePlatforms = [PlatformName]()
+            var defaultAvailableAvailabilities = [DefaultAvailability.ModuleAvailability]()
+
+            if let defaultAvailabilities = bundle.info.defaultAvailability?.modules[unifiedGraph.moduleName] {
+                // Availability platforms declared in the Info.plist as `unavailable`.
+                defaultUnconditionallyUnavailablePlatforms = defaultAvailabilities.filter {
+                    $0.state == .unavailable
+                }.map(\.platformName)
+                // Availability platforms declared in the Info.plist as `available`.
+                defaultAvailableAvailabilities = defaultAvailabilities.filter {
+                    $0.state != .unavailable
+                }
+            }
+            
+            // Platform names registered across the SGFs for each module.
+            let registeredPlatforms: [PlatformName] = unifiedGraph.moduleData.compactMap {
```

Comment:
I don't understand the word "registered" in this context. The information that this variable represents is the platforms that have specific symbol graph files. Would a name like `platformsFoundInSymbolGraphs` be clearer about what this is (maybe it doesn't even need that comment to explain what it is)?

```suggestion
            let platformsFoundInSymbolGraphs: [PlatformName] = unifiedGraph.moduleData.compactMap {
```

Reply:
https://github.com/apple/swift-docc/pull/807/commits/c55abc329895424275b96a3c47afef3f8c5bb3f9